sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlGroupConfigBuilder.java

```diff
@@ -166,6 +184,7 @@ public ThroughputControlGroupConfig build() {
                 this.groupName,
                 this.targetThroughput,
                 this.targetThroughputThreshold,
+                this.priorityLevel,
```

Comment:
currently, we require customers to set a targetThroughput or targetThroughputThreshold, else an `IllegalArgumentException` will be thrown during the validation step. 

however for priority throttling, my current understanding is priority level is the only required parameter, so we should also modify the checking above

Reply:
Yeah right. That's the reason we have exposed `ThroughputControlGroupConfig.priorityGroupConfig` which internally sets the throughput thresholds. If the customer uses the builder they'll still need to set the targetThroughput or targetThroughputThreshold. Open to changing it to the way you are suggesting. @FabianMeiswinkel your thoughts?

Reply:
ah I see, sorry missed that we are setting the threshold, so please ignore the above statement